annotation/src/main/resources/application.properties.EXAMPLE

```diff
@@ -1,5 +1,6 @@
 vep.url=http://grch37.rest.ensembl.org/vep/human/hgvs/VARIANT?content-type=application/json&xref_refseq=1&ccds=1&canonical=1&domains=1&hgvs=1&numbers=1&protein=1
 hotspots.url=http://cancerhotspots.org/api/hotspots/single/
-vep.overrides_uris=uniprot=isoform_overrides_uniprot.txt,mskcc=isoform_overrides_at_mskcc.txt
+vep.overrides_uris=uniprot:isoform_overrides_uniprot.txt,mskcc:isoform_overrides_at_mskcc.txt
```

Comment:
I was thinking the possibility of getting the content from an external URL such as https://raw.githubusercontent.com/mskcc/vcf2maf/master/data/isoform_overrides_at_mskcc. But maybe `override_sources` would be a better naming choice.